SmokeTest StandalonBrokerIT is flkay

**Summary**

- How often does the test fail? A couple of times already
- Does it block your work? PR merge is failing frequently
- Do we suspect that it is a real failure? No

**Failures**

```
Condition with alias 'until broker is ready' didn't complete within 10 seconds because assertion condition defined as a io.camunda.zeebe.it.smoke.StandaloneBrokerIT 1 expectation failed.
Expected status code <204> but was <503>.
.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
	at io.camunda.zeebe.it.smoke.StandaloneBrokerIT.beforeEach(StandaloneBrokerIT.java:52)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.AssertionError: 1 expectation failed.
Expected status code <204> but was <503>.
```

**Hypotheses**

The default timeout of 10s was not enough for Broker startup. Reasons could be slow node, but not sure. Doesn't seem to be a bug in Zeebe. Proposed fix is to increase the timeout in the test.

**Logs**

<details><summary>Logs</summary>
<pre>
14:28:10.169 [] [main] WARN  io.camunda.zeebe.test.util - No system property 'testMavenId' set, using default value 0
14:28:10.169 [] [main] INFO  io.camunda.zeebe.test.util.SocketUtil - Starting socket assignment with testForkNumber 0 and testMavenId 0
14:28:10.294 [] [main] INFO  io.camunda.zeebe.test.util.SocketUtil - Choosing next port 1025 for test fork 0 with range PortRange{host='localhost', basePort=1025, maxOffset=100, currentOffset=1}
14:28:10.294 [] [main] INFO  io.camunda.zeebe.test.util.SocketUtil - Choosing next port 1026 for test fork 0 with range PortRange{host='localhost', basePort=1025, maxOffset=100, currentOffset=2}
14:28:10.294 [] [main] INFO  io.camunda.zeebe.test.util.SocketUtil - Choosing next port 1027 for test fork 0 with range PortRange{host='localhost', basePort=1025, maxOffset=100, currentOffset=3}
14:28:10.294 [] [main] INFO  io.camunda.zeebe.test.util.SocketUtil - Choosing next port 1028 for test fork 0 with range PortRange{host='localhost', basePort=1025, maxOffset=100, currentOffset=4}

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v3.1.3)

14:28:13.057 [] [main] INFO  io.camunda.zeebe.it.smoke.StandaloneBrokerIT - Starting StandaloneBrokerIT using Java 17.0.8 with PID 5536 (started by runneradmin in D:\a\zeebe\zeebe\qa\integration-tests)
14:28:13.057 [] [main] DEBUG io.camunda.zeebe.it.smoke.StandaloneBrokerIT - Running with Spring Boot v3.1.3, Spring v6.0.11
14:28:13.057 [] [main] INFO  io.camunda.zeebe.it.smoke.StandaloneBrokerIT - The following 1 profile is active: "test"
14:28:18.857 [] [main] INFO  io.camunda.zeebe.broker.system - Started with development/test mode; data will be deleted on shutdown
14:28:20.139 [] [main] INFO  org.springframework.boot.actuate.endpoint.web.EndpointLinksResolver - Exposing 10 endpoint(s) beneath base path '/actuator'
14:28:24.737 [] [main] INFO  org.springframework.boot.web.embedded.netty.NettyWebServer - Netty started on port 52358
14:28:24.752 [] [main] INFO  io.camunda.zeebe.it.smoke.StandaloneBrokerIT - Started StandaloneBrokerIT in 14.381 seconds (process running for 50.688)
14:28:24.768 [] [main] WARN  io.camunda.zeebe.broker.system - No backup store is configured. Backups will not be taken
14:28:24.830 [] [main] INFO  io.camunda.zeebe.broker.system - Version: 8.3.0-SNAPSHOT
14:28:25.293 [] [main] INFO  io.camunda.zeebe.broker.system - Starting broker 0 with configuration {
  "network" : {
    "host" : "0.0.0.0",
    "portOffset" : 0,
    "maxMessageSize" : "4MB",
    "advertisedHost" : "0.0.0.0",
    "commandApi" : {
      "host" : "0.0.0.0",
      "port" : 1026,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 1026,
      "address" : "0.0.0.0:1026",
      "advertisedAddress" : "0.0.0.0:1026"
    },
    "internalApi" : {
      "host" : "0.0.0.0",
      "port" : 1027,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 1027,
      "address" : "0.0.0.0:1027",
      "advertisedAddress" : "0.0.0.0:1027"
    },
    "security" : {
      "enabled" : false,
      "certificateChainPath" : null,
      "privateKeyPath" : null
    },
    "maxMessageSizeInBytes" : 4194304
  },
  "cluster" : {
    "initialContactPoints" : [ ],
    "partitionIds" : [ 1 ],
    "nodeId" : 0,
    "partitionsCount" : 1,
    "replicationFactor" : 1,
    "clusterSize" : 1,
    "clusterName" : "zeebe-cluster",
    "heartbeatInterval" : "PT0.25S",
    "electionTimeout" : "PT2.5S",
    "membership" : {
      "broadcastUpdates" : false,
      "broadcastDisputes" : true,
      "notifySuspect" : false,
      "gossipInterval" : "PT0.25S",
      "gossipFanout" : 2,
      "probeInterval" : "PT1S",
      "probeTimeout" : "PT0.1S",
      "suspectProbes" : 3,
      "failureTimeout" : "PT10S",
      "syncInterval" : "PT10S"
    },
    "raft" : {
      "enablePriorityElection" : true,
      "flush" : {
        "enabled" : true,
        "delayTime" : "PT0S"
      }
    },
    "messageCompression" : "NONE"
  },
  "threads" : {
    "cpuThreadCount" : 2,
    "ioThreadCount" : 2
  },
  "data" : {
    "directory" : "C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\zeebe7957564021468670155\\data",
    "runtimeDirectory" : null,
    "logSegmentSize" : "128MB",
    "snapshotPeriod" : "PT5M",
    "logIndexDensity" : 100,
    "disk" : {
      "enableMonitoring" : true,
      "monitoringInterval" : "PT1S",
      "freeSpace" : {
        "processing" : "2048MB",
        "replication" : "1024MB"
      }
    },
    "backup" : {
      "store" : "NONE",
      "s3" : {
        "bucketName" : null,
        "endpoint" : null,
        "region" : null,
        "accessKey" : null,
        "secretKey" : null,
        "apiCallTimeout" : "PT3M",
        "forcePathStyleAccess" : false,
        "compression" : null,
        "basePath" : null
      },
      "gcs" : {
        "bucketName" : null,
        "basePath" : null,
        "host" : null,
        "auth" : "AUTO"
      }
    },
    "logSegmentSizeInBytes" : 134217728
  },
  "exporters" : { },
  "gateway" : {
    "network" : {
      "host" : "0.0.0.0",
      "port" : 1025,
      "minKeepAliveInterval" : "PT30S",
      "maxMessageSize" : "4MB"
    },
    "cluster" : {
      "initialContactPoints" : [ "0.0.0.0:1027" ],
      "requestTimeout" : "PT15S",
      "clusterName" : "zeebe-cluster",
      "memberId" : "gateway",
      "host" : "0.0.0.0",
      "advertisedHost" : "0.0.0.0",
      "port" : 26502,
      "advertisedPort" : 26502,
      "membership" : {
        "broadcastUpdates" : false,
        "broadcastDisputes" : true,
        "notifySuspect" : false,
        "gossipInterval" : "PT0.25S",
        "gossipFanout" : 2,
        "probeInterval" : "PT1S",
        "probeTimeout" : "PT0.1S",
        "suspectProbes" : 3,
        "failureTimeout" : "PT10S",
        "syncInterval" : "PT10S"
      },
      "security" : {
        "enabled" : false,
        "certificateChainPath" : null,
        "privateKeyPath" : null,
        "authentication" : {
          "mode" : "NONE",
          "identity" : {
            "issuerBackendUrl" : null,
            "audience" : "zeebe-api",
            "type" : "KEYCLOAK"
          }
        }
      },
      "messageCompression" : "NONE"
    },
    "threads" : {
      "managementThreads" : 1,
      "grpcMinThreads" : 2,
      "grpcMaxThreads" : 4
    },
    "security" : {
      "enabled" : false,
      "certificateChainPath" : null,
      "privateKeyPath" : null,
      "authentication" : {
        "mode" : "NONE",
        "identity" : {
          "issuerBackendUrl" : null,
          "audience" : "zeebe-api",
          "type" : "KEYCLOAK"
        }
      }
    },
    "longPolling" : {
      "enabled" : true
    },
    "interceptors" : [ ],
    "enable" : true
  },
  "backpressure" : {
    "enabled" : true,
    "algorithm" : "AIMD",
    "aimd" : {
      "requestTimeout" : "PT0.2S",
      "initialLimit" : 100,
      "minLimit" : 1,
      "maxLimit" : 1000,
      "backoffRatio" : 0.9
    },
    "fixed" : {
      "limit" : 20
    },
    "vegas" : {
      "alpha" : 3,
      "beta" : 6,
      "initialLimit" : 20
    },
    "gradient" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0
    },
    "gradient2" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0,
      "longWindow" : 600
    }
  },
  "experimental" : {
    "maxAppendsPerFollower" : 2,
    "maxAppendBatchSize" : "32KB",
    "disableExplicitRaftFlush" : false,
    "rocksdb" : {
      "columnFamilyOptions" : { },
      "enableStatistics" : false,
      "memoryLimit" : "512MB",
      "maxOpenFiles" : -1,
      "maxWriteBufferNumber" : 6,
      "minWriteBufferNumberToMerge" : 3,
      "ioRateBytesPerSecond" : 0,
      "disableWal" : true,
      "enableSstPartitioning" : true
    },
    "raft" : {
      "requestTimeout" : "PT2.5S",
      "snapshotRequestTimeout" : "PT2.5S",
      "maxQuorumResponseTimeout" : "PT0S",
      "minStepDownFailureCount" : 3,
      "preferSnapshotReplicationThreshold" : 100,
      "preallocateSegmentFiles" : true
    },
    "partitioning" : {
      "scheme" : "ROUND_ROBIN",
      "fixed" : [ ]
    },
    "queryApi" : {
      "enabled" : false
    },
    "consistencyChecks" : {
      "enablePreconditions" : false,
      "enableForeignKeyChecks" : false,
      "settings" : {
        "enablePreconditions" : false,
        "enableForeignKeyChecks" : false
      }
    },
    "engine" : {
      "messages" : {
        "ttlCheckerBatchLimit" : 2147483647,
        "ttlCheckerInterval" : "PT1M"
      },
      "caches" : {
        "drgCacheCapacity" : 1000
      }
    },
    "features" : {
      "enableYieldingDueDateChecker" : true,
      "enableActorMetrics" : false,
      "enableMessageTtlCheckerAsync" : false,
      "enableTimerDueDateCheckerAsync" : false,
      "enableStraightThroughProcessingLoopDetector" : true,
      "enableDynamicClusterTopology" : false
    },
    "maxAppendBatchSizeInBytes" : 32768
  },
  "executionMetricsExporterEnabled" : false,
  "processing" : {
    "maxCommandsInBatch" : 100,
    "enableAsyncScheduledTasks" : true
  }
}
14:28:25.293 [Startup] [zb-actors-1] DEBUG io.camunda.zeebe.broker.system - Startup was called with context: io.camunda.zeebe.broker.bootstrap.BrokerStartupContextImpl@683d049f
14:28:25.293 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Cluster Services (Start)
14:28:25.324 [] [netty-messaging-event-nio-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:1027
14:28:25.887 [Startup] [zb-actors-1] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started messaging service bound to [0.0.0.0:1027], advertising 0.0.0.0:1027, and using plaintext
14:28:28.372 [] [netty-unicast-event-nio-client-0] INFO  io.atomix.cluster.messaging.impl.NettyUnicastService - Started plaintext unicast service bound to 0.0.0.0:1027, advertising 0.0.0.0:1027
14:28:28.372 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Local node Node{id=0, address=0.0.0.0:1027} joined the bootstrap service
14:28:28.372 [] [atomix-cluster-0] DEBUG io.atomix.cluster.protocol.swim - Nodes from discovery service []
14:28:28.383 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.swim - Started
14:28:28.383 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Started cluster membership service for member Member{id=0, address=0.0.0.0:1027, properties={}}
14:28:28.383 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Started
14:28:28.383 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Started
14:28:28.383 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Cluster Topology Manager
14:28:28.383 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Disk Space Usage Monitor
14:28:28.383 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Health Monitor
14:28:28.383 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Broker Admin Interface
14:28:28.399 [Startup] [zb-actors-0] INFO  io.camunda.zeebe.broker.system - Startup API Messaging Service
14:28:28.399 [] [netty-messaging-event-nio-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:1026
14:28:28.399 [] [netty-messaging-event-nio-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started messaging service bound to [0.0.0.0:1026], advertising 0.0.0.0:1026, and using plaintext
14:28:28.399 [Startup] [zb-actors-0] DEBUG io.camunda.zeebe.broker.system - Bound API to [0.0.0.0:1026], using advertised address 0.0.0.0:1026 
14:28:28.399 [Startup] [zb-actors-0] INFO  io.camunda.zeebe.broker.system - Startup Broker Transport
14:28:28.414 [Startup] [zb-actors-0] INFO  io.camunda.zeebe.broker.system - Startup Command API
14:28:28.430 [Startup] [zb-actors-0] INFO  io.camunda.zeebe.broker.system - Startup Embedded Gateway
14:28:28.696 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup JobStreamService
14:28:28.712 [Startup] [zb-actors-1] INFO  io.camunda.zeebe.broker.system - Startup Partition Manager
14:28:28.727 [] [Thread-14] INFO  io.camunda.zeebe.broker.partitioning.PartitionManagerImpl - Starting partitions
14:28:28.743 [Startup] [zb-actors-0] DEBUG io.camunda.zeebe.broker.system - Finished startup process
14:28:28.790 [] [Thread-15] DEBUG io.camunda.zeebe.broker.partitioning.bootstrap - Starting raft partition 1
14:28:28.790 [] [Thread-15] INFO  io.atomix.raft.partition.impl.RaftPartitionServer - RaftPartitionServer{raft-partition-partition-1} - Starting server for partition PartitionId{id=1, group=raft-partition}
14:28:28.915 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.swim - 0 - Detected local properties change {brokerInfo=EADJAAAAAwAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkMAAAAMC4wLjAuMDoxMDI2BQAADAAADgAAADguMy4wLVNOQVBTSE9UBQAA, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}
14:28:28.915 [GatewayTopologyManager] [zb-actors-1] DEBUG io.camunda.zeebe.gateway - Received metadata change from Broker 0, partitions {}, terms {} and health {}.
14:28:28.930 [GatewayTopologyManager] [zb-actors-0] DEBUG io.camunda.zeebe.gateway - Received metadata change from Broker 0, partitions {}, terms {} and health {}.
14:28:28.979 [] [Thread-15] DEBUG io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Priority election is enabled with target priority 1 and node priority 1
</pre>
</details>

--- a/qa/integration-tests/src/test/java/io/camunda/zeebe/it/smoke/StandaloneBrokerIT.java
+++ b/qa/integration-tests/src/test/java/io/camunda/zeebe/it/smoke/StandaloneBrokerIT.java
@@ -21,6 +21,7 @@ import io.camunda.zeebe.test.util.asserts.TopologyAssert;
 import io.restassured.builder.RequestSpecBuilder;
 import io.restassured.http.ContentType;
 import io.restassured.specification.RequestSpecification;
+import java.time.Duration;
 import java.util.Objects;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.extension.ExtendWith;
@@ -49,7 +50,9 @@ final class StandaloneBrokerIT {
 
   @BeforeEach
   void beforeEach() {
-    await("until broker is ready").untilAsserted(this::assertBrokerIsReady);
+    await("until broker is ready")
+        .timeout(Duration.ofSeconds(60)) // A slow ci infrastructure can result in slow startup
+        .untilAsserted(this::assertBrokerIsReady);
   }
 
   /** A simple smoke test to ensure the broker starts and can perform basic functionality. */

